packages/sdk/src/constants/gas-controllers/moe-nft-mint.ts

```diff
@@ -0,0 +1,154 @@
+  },
+  [networks.binanceSmartChainTestnet.chain]: {
+    gasPrice: undefined,
+    gasLimit: undefined,
+    gasLimitMultiplier: 110,
+    gasPriceMultiplier: 110,
+  },
+  [networks.avalanche.chain]: {
+    gasPrice: undefined,
+    gasLimit: undefined,
+    gasLimitMultiplier: 110,
+    gasPriceMultiplier: 110,
+  },
+  [networks.avalancheTestnet.chain]: {
+    gasPrice: undefined,
+    gasLimit: undefined,
+    gasLimitMultiplier: 110,
+    gasPriceMultiplier: 110,
+  },
+  [networks.optimism.chain]: {
+    gasPrice: undefined,
+    gasLimit: undefined,
+    gasLimitMultiplier: 110,
+    gasPriceMultiplier: 110,
+  },
+  [networks.optimismTestnetGoerli.chain]: {
+    gasPrice: undefined,
+    gasLimit: undefined,
+    gasLimitMultiplier: 110,
+    gasPriceMultiplier: 110,
+  },
+  [networks.optimismTestnetSepolia.chain]: {
+    gasPrice: undefined,
+    gasLimit: undefined,
+    gasLimitMultiplier: 110,
+    gasPriceMultiplier: 110,
+  },
+  [networks.arbitrumOne.chain]: {
+    gasPrice: undefined,
+    gasLimit: undefined,
+    gasLimitMultiplier: 110,
+    gasPriceMultiplier: 110,
+  },
+  [networks.arbitrumTestnetGoerli.chain]: {
+    gasPrice: undefined,
+    gasLimit: undefined,
+    gasLimitMultiplier: 110,
+    gasPriceMultiplier: 110,
+  },
+  [networks.arbitrumTestnetSepolia.chain]: {
+    gasPrice: undefined,
+    gasLimit: undefined,
+    gasLimitMultiplier: 110,
+    gasPriceMultiplier: 110,
+  },
+  [networks.mantle.chain]: {
+    gasPrice: undefined,
+    gasLimit: undefined,
+    gasLimitMultiplier: 110,
+    gasPriceMultiplier: 110,
+  },
+  [networks.mantleTestnet.chain]: {
+    gasPrice: undefined,
+    gasLimit: undefined,
+    gasLimitMultiplier: 110,
+    gasPriceMultiplier: 110,
+  },
+  [networks.zora.chain]: {
+    gasPrice: '100000000',
+    gasLimit: undefined,
+    gasLimitMultiplier: 110,
+    gasPriceMultiplier: 110,
+  },
+  [networks.zoraTestnetGoerli.chain]: {
+    gasPrice: undefined,
+    gasLimit: undefined,
+    gasLimitMultiplier: 110,
+    gasPriceMultiplier: 110,
+  },
+  [networks.zoraTestnetSepolia.chain]: {
+    gasPrice: undefined,
+    gasLimit: undefined,
+    gasLimitMultiplier: 110,
+    gasPriceMultiplier: 110,
+  },
+  [networks.base.chain]: {
+    gasPrice: undefined,
+    gasLimit: undefined,
+    gasLimitMultiplier: 110,
+    gasPriceMultiplier: 110,
+  },
+  [networks.baseTestnetGoerli.chain]: {
+    gasPrice: undefined,
+    gasLimit: undefined,
+    gasLimitMultiplier: 110,
+    gasPriceMultiplier: 110,
+  },
+  [networks.baseTestnetSepolia.chain]: {
+    gasPrice: undefined,
+    gasLimit: undefined,
+    gasLimitMultiplier: 110,
+    gasPriceMultiplier: 110,
+  },
+  [networks.localhost.chain]: {
+    gasPrice: undefined,
+    gasLimit: undefined,
+    gasLimitMultiplier: 110,
+    gasPriceMultiplier: 110,
+  },
+  [networks.localhost2.chain]: {
+    gasPrice: undefined,
+    gasLimit: undefined,
+    gasLimitMultiplier: 110,
+    gasPriceMultiplier: 110,
+  },
+}
```

Comment:
add `as const satisfies Record<SupportedChainIds, GasParams>`  like i did in this pr (https://github.com/holographxyz/holograph-app/pull/937/files)